src/modules/dream/geometry/HeimdalMantle.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2022 - 2023 European Spallation Source, see LICENSE file
+//===----------------------------------------------------------------------===//
+///
+/// \file
+///
+/// \brief CDT Mantle module abstractions
+///
+/// Consult ICD for logical geometry dimensions, rotations etc.
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <common/debug/Trace.h>
+#include <dream/geometry/Config.h>
+#include <dream/readout/DataParser.h>
+#include <logical_geometry/ESSGeometry.h>
+
+// #undef TRC_LEVEL
+// #define TRC_LEVEL TRC_L_DEB
+
+namespace Dream {
+
+class HeimdalMantle {
+public:
+  const uint8_t WiresPerCounter{32};
```

Comment:
Why public? As I see it's not refered from anywhere else. But If you thin kthis should be a public const then maybe it should be static for the class.